cloud/getting-started/cc-trial.md

```diff
@@ -0,0 +1,36 @@
+---
+id: cc-trial
+title: About the Centreon Cloud trial
+---
+
+## What is the Centreon Cloud trial?
+
+Centreon Cloud trial is a **free 7-days license** that includes the features available with our [IT Edition](https://www.centreon.com/centreon-editions/#editions-table) and with a monitoring capacity of up to 125 hosts.
+
+Centreon Cloud offers the Centreon monitoring platform in a SaaS version, for an experience free of installation and maintenance.
+
+> Visit the [Editions](https://www.centreon.com/centreon-editions/) page to find out about the features included in the various editions of Centreon.
+
+## About users and organizations
+
+### User accounts information
+
+User accounts are managed in the [Centreon CIAM](../ciam/ciam.md) (Customer Identity and Access Management) module: the Centreon’s access management tool. Your administrator needs to create an organization and invite the other users to it so that all team members can work on the same Centreon platform. 
```

Comment:
```suggestion
User accounts are managed in the [Centreon CIAM](../ciam/ciam.md) (Customer Identity and Access Management) module, Centreon’s access management tool. Your administrator needs to create an organization and invite the other users to it so that all team members can work on the same Centreon platform. 
```